server/db/driver/pg/internal/matches.go

```diff
@@ -117,6 +117,19 @@ const (
 	WHERE (takerAccount = $1 OR makerAccount = $1)
 		AND active;`
 
+	RetrieveMarketMatches = `SELECT matchid, active, takerSell,
+		takerOrder, takerAccount, takerAddress,
+		makerOrder, makerAccount, makerAddress,
+		epochIdx, epochDur, quantity, rate, baseRate, quoteRate, status
+	FROM %s;`
+
+	RetrieveActiveMarketMatches = `SELECT matchid, takerSell,
+		takerOrder, takerAccount, takerAddress,
+		makerOrder, makerAccount, makerAddress,
+		epochIdx, epochDur, quantity, rate, baseRate, quoteRate, status
+	FROM %s
+	Where active;`
```

Comment:
For both of these we should exclude cancel order matches with `WHERE takerSell IS NOT NULL`

Reply:
Best to sort as well, `ORDER BY epochIdx * epochDur DESC`